recipes/espsim/setup-py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
+index 2d0ca17..9c35f21 100644
+--- a/setup.py
++++ b/setup.py
+@@ -5,7 +5,7 @@ with open('README.md', encoding='utf-8') as f:
+ 
+ setup(
+     name='espsim',
+-    version='0.0.1',
++    version='1.0.0',
```

Comment:
This is an odd patch. Can you add a comment on why that is needed?

Reply:
Simply because upstream didnt update that field when tagging 1.0. I added a note in the recipe.